mava/wrappers/jumanji.py

```diff
@@ -265,3 +267,103 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             return specs.Spec(ObservationGlobalState, "ObservationSpec", **obs_data)
 
         return specs.Spec(Observation, "ObservationSpec", **obs_data)
+
+
+class CleanerWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Cleaner environment."""
+
+    def __init__(self, env: Cleaner, add_global_state: bool = False):
+        super().__init__(env, add_global_state)
+        self._env: Cleaner
+
+    def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
+        """Modify the timestep for the Cleaner environment."""
+
+        def create_agents_view(grid: chex.Array, agents_locations: chex.Array) -> chex.Array:
+            """Create separate channels for dirty cells, wall cells and agent positions.
+            Also add a channel that marks an agent's own position."""
+
+            num_agents = self.num_agents
+
+            # A: Number of agents
+            # R: Number of grid rows
+            # C: Number of grid columns
+            # grid: (R, C)
+            # agents_locations: (A, 2)
+
+            # Get dirty / wall tiles from first agent's obs and tile in agents dimension.
+            # (A, R, C)
+            dirty_channel = jnp.tile(grid == DIRTY, (num_agents, 1, 1))
+            wall_channel = jnp.tile(grid == WALL, (num_agents, 1, 1))
```

Comment:
It is a bit easier to read when the a shape is on the same line.
```suggestion
            dirty_channel = jnp.tile(grid == DIRTY, (num_agents, 1, 1))  # (A, R, C)
            wall_channel = jnp.tile(grid == WALL, (num_agents, 1, 1))  # (A, R, C)
```

Reply:
On some of these lines, putting the common on the same line causes the precommit to to turn it into 3 lines. Is this still preferable? Example:
![image](https://github.com/instadeepai/Mava/assets/90381208/cc535ebe-31d0-4354-87b7-f10ece141aaa)

Reply:
Yep, this is fine.